src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -38,4 +44,13 @@ public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto subur
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    @GetMapping("/{id}/orders")
+    public List<OpenOrderResponseDto> getOpenOrders(@PathVariable Long id, @RequestParam @NotNull String status){
```

Comment:
What's the status used for?

Reply:
status means the order status, value "open" is for the ordered to be accepted/rejected by franchisee
`/{id}/orders?status=open` is the link to use.